### PR TITLE
PM-10052: Add remaining carousel pages

### PR DESCRIPTION
**Pull request by matt-livefront**
## 🎟️ Tracking

[PM-10052](https://bitwarden.atlassian.net/browse/PM-10052): Display Carousel 2 - Quick and Easy Login
[PM-10053](https://bitwarden.atlassian.net/browse/PM-10053): Display Carousel 3 - Level Up Your Logins
[PM-10054](https://bitwarden.atlassian.net/browse/PM-10054): Display Carousel 4 - Your Data, When and Where You Need It

## 📔 Objective

Adds the remaining pages to the intro carousel.

## 📸 Screenshots

https://github.com/user-attachments/assets/408d9c9f-3eef-4556-b4b1-b1ee96a06c07

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

[PM-10052]: https://bitwarden.atlassian.net/browse/PM-10052?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[PM-10053]: https://bitwarden.atlassian.net/browse/PM-10053?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[PM-10054]: https://bitwarden.atlassian.net/browse/PM-10054?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: BitwardenShared/UI/Auth/IntroCarousel/__Snapshots__/IntroCarouselViewTests/test_snapshot_introCarousel_page1.3.png
Comment (fedemkr) [inline BitwardenShared/UI/Auth/IntroCarousel/__Snapshots__/IntroCarouselViewTests/test_snapshot_introCarousel_page1.3.png:1]: 🎨 It seems that the index `dots` are overlapping the text.

**Review by fedemkr — COMMENTED** [review]
Looks good just one small comment.

File: BitwardenShared/UI/Auth/IntroCarousel/__Snapshots__/IntroCarouselViewTests/test_snapshot_introCarousel_page1.3.png
Comment (matt-livefront) [reply BitwardenShared/UI/Auth/IntroCarousel/__Snapshots__/IntroCarouselViewTests/test_snapshot_introCarousel_page1.3.png:1]: I switched the page indicator dots to use the built-in background style to match what the authenticator app is doing. This looks correct on a device or simulator, but doesn't appear to render for snapshots.

Screenshot 2024-08-05 at 9 45 43 AM

Comment (fedemkr) [reply BitwardenShared/UI/Auth/IntroCarousel/__Snapshots__/IntroCarouselViewTests/test_snapshot_introCarousel_page1.3.png:1]: Alright if on-device looks good then we're fine 😄 Thanks for checking into it.

**Review by fedemkr — APPROVED** [review]

**Pull request merged**